test/test.js

```diff
@@ -144,6 +145,7 @@ describe('transit', function() {
 
       var roundTrip = recordTransit.fromJSON(recordTransit.toJSON(data));
       expect(roundTrip).to.eql(data);
+      expect(roundTrip.toString()).to.eql(data.toString());
     });
 
     it('should serialize unspecified Record as a Map', function() {
```

Comment:
Could you add the toString here as well please?

I wonder if it's worth writing a helper method / chai-plugin (inline, not as separate module) and using that for every comparison.